packages/components/src/checkbox-control/style.scss

```diff
@@ -5,6 +5,20 @@
 	@include break-small() {
 		--checkbox-input-size: 20px;
 	}
+
+	.components-base-control__help {
+		margin-left: calc(var(--checkbox-input-size) + #{$grid-unit-15});
+	}
```

Comment:
This is inevitable at the moment, but raised as an issue at #60836.

Reply:
Saw the linked issue. It probably wouldn't be as flexible, but could checkbox control do something like this to indent the text?
```jsx
help={ <span="components-checkbox-control__help-text">{ helpText }</span> }
```

```css
.components-checkbox-control__help {
    margin-left: calc(var(--checkbox-input-size) + #{$grid-unit-15});
}
```

Reply:
Oh, good one! Yes that could work in this instance, with a `display: inline-block` to account for when the help text is long enough to wrap.

Reply:
I'm not sure I understand the benefit to doing this? Feel free to push any changes :)

Reply:
Good question. The point is that we shouldn't be using selectors from other components, because they aren't guaranteed to be stable. Ideally they would all be obfuscated/randomized so they aren't even available outside the original component.

I'll try it out and push the changes 👍

Reply:
Done in 8f08d50b7941fda13aeaf65f5c70921d1bae4659